deploy/helm/sumologic/templates/setup/setup-job.yaml

```diff
@@ -16,19 +16,30 @@ spec:
     spec:
       restartPolicy: OnFailure
       serviceAccountName: {{ template "sumologic.fullname" . }}-setup
+      volumes:
+      - name: setup
+        configMap:
+          name: {{ template "sumologic.fullname" . }}-setup
+          defaultMode: 0777
       containers:
+      - name:  kubectl
+        image: gcr.io/google_containers/kubectl:v1.0.7
+        command: ["/kubectl"]
+        args: ["proxy", "-p", "8001"]
```

Comment:
Need to ask @frankreno about potential security concern on exposing a port in the container. Is there other ways to initialize the `kubernetes` provider?

Reply:
That was only intended to be for the demo I have you guys.  We will need to support proper authentication and see if we can do this - https://www.terraform.io/docs/providers/kubernetes/index.html#authentication